helm-charts/templates/data-plane/gateway-components/gateway-runtime/gateway-runtime-deployment.yaml

```diff
@@ -94,7 +94,7 @@ spec:
             {{- if and .Values.wso2.apk.metrics .Values.wso2.apk.metrics.enabled }}
               value: -Dhttpclient.hostnameVerifier=AllowAll -Xms512m -Xmx512m -XX:MaxRAMFraction=2 -Dapk.jmx.metrics.enabled=true -javaagent:/home/wso2/lib/jmx_prometheus_javaagent-0.20.0.jar=18006:/tmp/metrics/prometheus-jmx-config-enforcer.yml
             {{- else }}
-              value: -Dhttpclient.hostnameVerifier=AllowAll -Xms512m -Xmx512m -XX:MaxRAMFraction=2
+              value: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5006 -Dhttpclient.hostnameVerifier=AllowAll -Xms512m -Xmx512m -XX:MaxRAMFraction=2
```

Comment:
let's remove this debugging line

Reply:
Fixed in https://github.com/wso2/apk/pull/2178/commits/d56240231c37bd8971d4e55b8c4f06cd234f9308